resources/server/discord.ts

```diff
@@ -37,3 +38,28 @@ export async function reportTweetToDiscord(tweet: Tweet, reportingProfile: Profi
 
   return await axios.post(POST_CHANNEL_URL, { content }, { headers: DISCORD_HEADERS });
 }
+
+export async function reportListingToDiscord(
```

Comment:
Just want to make sure this has been tested, I don't think I've run the twitter version for a few months. Although it should work.